File: tests/script/examples/primes.js
function is_prime(n)
{
    // TEST: convert to number
    n = new Number(n);
    if(n < 2) return false;
    let i = 2;
    let c = 0;
    while(i < n)
    {
        if(n % i == 0)
            c++;
        i++;
    }
    return c == 0;
}

let i = 2000;
let start = new Date();
while(i < 2400) console.log(i, ": ", is_prime(new String(i++)));
console.log("Time: " + (new Date().getTime() - start.getTime()) + "s");

File: tests/script/examples/primes.py
import time

def is_prime(n):
    # TEST: convert to number
    n = int(n)
    if n < 2:
        return False
    i = 2
    c = 0
    while i < n:
        if n % i == 0:
            c += 1
        i += 1
    return c == 0

i = 2000
start = time.time()
while i < 2400:
    print(str(i) + ": " + str(is_prime(str(i))))
    i += 1
print("Time: " + str(time.time() - start) + "s")
